aspnetcore/blazor/supported-platforms.md

```diff
@@ -5,40 +5,58 @@ description: Learn about the supported platforms for ASP.NET Core Blazor.
 monikerRange: '>= aspnetcore-3.1'
 ms.author: riande
 ms.custom: mvc
-ms.date: 05/19/2020
+ms.date: 10/01/2020
 no-loc: ["ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
 uid: blazor/supported-platforms
 ---
 # ASP.NET Core Blazor supported platforms
 
 By [Luke Latham](https://github.com/guardrex)
 
-## Browser requirements
+::: moniker range=">= aspnetcore-5.0"
 
```

Comment:
```suggestion

Blazor Server and Blazor WebAssembly are supported in the following browsers:

```

Reply:
I'll make a minor edit. MS style is no colon before a table. They just want a period.